README.md

```diff
@@ -14,9 +27,14 @@ docker pull pgc-images.sbgenomics.com/zhuangzhuanggeng/d3b_hope_analysis:latest
 
 To run the docker, run the command line below. For mac M1 user, add `--platform=linux/arm64`.
 ```
-docker run -d -e PASSWORD=pass -p 8787:8787 --name <CONTAINER_NAME> -v $PWD:/home/rstudio/HOPE pgc-images.sbgenomics.com/zhuangzhuanggeng/d3b_hope_analysis:latest
+docker run -d -e PASSWORD=pass -p 8787:8787 --name <CONTAINER_NAME> -v $PWD:/home/rstudio/hope-cohort-analysis pgc-images.sbgenomics.com/zhuangzhuanggeng/d3b_hope_analysis:latest
+```
 
+## Navigate to the repository root
 ```
+cd /home/rstudio/hope-cohort-analysis
+```
+
 
 ## Modules
 
```

Comment:
"7) `tmb-calculation`: Adapted from [OpenPedCan-anaysis](https://github.com/d3b-center/OpenPedCan-analysis/tree/dev/analyses/tmb-calculation)"
This line refers to something I think no longer exists and is not in the list of modules.